org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/handler/ZigBeeThingHandler.java

```diff
@@ -423,6 +439,23 @@ private synchronized void doNodeInitialisation() {
         coordinatorHandler.serializeNetwork();
     }
 
+    private int findMinimumPollOrReportingInterval(Map<ChannelUID, ZigBeeBaseChannelConverter> channels) {
```

Comment:
Likewise - I think this name is a mouthful and not so easy to understand.  How about just ```getExpectedUpdatePeriod``` or ```getRefreshPeriod```?

Reply:
See above, `getExpectedUpdatePeriod` sounds good to me as well.